### PR TITLE
[MINI-4694] download files to device

### PR DESCRIPTION
**Pull request by laubengaier**
# Description
Add new interface to download files to device instead of XHR (+base64)

## Links
https://jira.rakuten-it.com/jira/browse/MINI-4694

# Checklist
- [ ] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

**Review by Climbatize — APPROVED** [review]
LGTM

File: js-miniapp-bridge/src/common-bridge.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { Contact } from './types/contact';
 import { MessageToContact } from './types/message-to-contact';
 import { Points } from './types/points';
 import { HostEnvironmentInfo } from './types/host-environment-info';
+import { DownloadHeaders } from './types/download-headers';
```
Comment (rleojoseph) [inline js-miniapp-bridge/src/common-bridge.ts:21]: May be DownloadFileHeaders 🤔

**Review by rleojoseph — CHANGES_REQUESTED** [review]
Overall looks ok to me. Could you please update the test cases for this functionality? We are already less than 80%

**Review by munir-rakuten — APPROVED** [review]
Looks good to me 👍

File: js-miniapp-bridge/src/common-bridge.ts (hunk 1 above)
Comment (laubengaier) [reply js-miniapp-bridge/src/common-bridge.ts:21]: @rleojoseph do you mean a rename for the type overall?

**Review by rleojoseph — APPROVED** [review]
Thank you for updating Testcases. Looks good to me :+1:

**Pull request merged**